static/app/utils/profiling/flamegraph.ts

```diff
@@ -214,9 +217,10 @@ export class Flamegraph {
       }
 
       frames.push(stackTop);
-      this.depth = Math.max(stackTop.depth, this.depth);
+      depth = Math.max(stackTop.depth, depth);
     };
 
+    this.depth = depth;
```

Comment:
Should this be set after the `profile.forEach`?

Reply:
Ups, yes it should